dissect/cstruct/cstruct.py

```diff
@@ -29,6 +29,7 @@
     Wchar,
     WcharArray,
```

Comment:
Can you add the LEB128 to `dissect/cstruct/types/__init__.py` and then add it to this import list?